code/packages/web/src/pages/ForgotPasswordPage.css

```diff
@@ -0,0 +1,97 @@
+/* Styles for the Forgot Password Page */
```

Comment:
We shouldn't use html element tags like `body`, `h2`, `form`, `label`, `input`, `button` etc because using them like this would change styling for all files and pages and we don't want that. We want to create styling for just the ForgotPasswordPage.tsx.
 
 To do that, wrap entire contents of this file under the class name which is being used at the root div of the component.

Currently, the root div has className forgot-password-container, so wrap this with that class like this:

```
.forgot-password-container {
  body {
    font-family: 'Arial', sans-serif;
    background: linear-gradient(to bottom, #e0e5f7, #cfd7f5);
    margin: 0;
    display: flex;
    justify-content: center;
    align-items: center;
    height: 100vh;
  }
  ... {other class here}
 }
```